code/game/gamemodes/changeling/powers/augmented_eyesight.dm

```diff
@@ -46,7 +43,7 @@
 
 /obj/item/organ/internal/cyberimp/eyes/shield/ling/on_life()
 	..()
-	if(owner.eye_blind || owner.eye_stat || owner.eye_blurry || (owner.disabilities & BLIND) || (owner.disabilities & NEARSIGHT))
+	if(owner.eye_blind>1 || (owner.eye_blind && owner.stat !=UNCONSCIOUS) || owner.eye_stat || owner.eye_blurry || (owner.disabilities & NEARSIGHT))
```

Comment:
Using "if(owner.eye_blind)" and "if(owner.eye_stat)" means you will get oculine if you just sleep.